### ISSUE TITLE
Windows ansible script fails at Windows Update step

### ISSUE THREAD

**Issue by lumpfish** [AdoptOpenJDK/openjdk-infrastructure#1469]
Running the ansible playbook on a newly created Windows Server 2012 R2 machine failed at the Windows Update step with:
```
TASK [Windows_Updates : Download and Install Windows Updates] ******************
fatal: [9.20.198.24]: FAILED! => {"changed": false, "filtered_updates": {}, "msg": "Failed to search for updates: Exception calling \"Search\" with \"1\" argument(s): \"Exception from HRESULT: 0x80072EE5\"", "updates": {}}
...ignoring

TASK [Windows_Updates : Reboot machine if necessary] ***************************
fatal: [9.20.198.24]: FAILED! => {"msg": "The conditional check 'update_result.reboot_required' failed. The error was: error while evaluating conditional (update_result.reboot_required): 'dict object' has no attribute 'reboot_required'\n\nThe error appears to be in '/disk2/git/openjdk-infrastructure/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Windows_Updates/tasks/main.yml': line 11, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Reboot machine if necessary\n  ^ here\n"}
```

**Comment by Willsparker** [comment]
I looked into this yesterday and I think this is an issue with the provisioning service that @lumpfish and I use to create the windows machines. It seems to be an intermittent issue as something can be done to make it work, for instance: Manually run windows update, checking for updates from the internet; opening Internet Explorer; Just waiting for awhile - though none of these consistently fix it.

However, the issue is replicated when you manually check for updates that are specifically managed by your system administrator. You're able to update the system from updates found online so if anyone else encounters this problem, I'd suggest doing that, rebooting and skipping the `Windows_Updates` task on the playbook. Closing issue
